src/main/java/com/fouristhenumber/utilitiesinexcess/common/blocks/BlockDrum.java

```diff
@@ -0,0 +1,227 @@
+package com.fouristhenumber.utilitiesinexcess.common.blocks;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockContainer;
+import net.minecraft.block.material.Material;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.item.EntityItem;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemBlock;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.ChatComponentText;
+import net.minecraft.util.StatCollector;
+import net.minecraft.world.World;
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.fluids.IFluidContainerItem;
+
+import com.fouristhenumber.utilitiesinexcess.common.tileentities.TileEntityDrum;
+
+public class BlockDrum extends BlockContainer {
+
+    int capacity = 16000;
```

Comment:
```suggestion
    final int capacity = 16000;
```
It can also be changed later to just `final int capacity;` and set in the constructor when we want to add more drums with more capacity options